.github/workflows/test_all_models.py

```diff
@@ -0,0 +1,87 @@
+import onnx
+from pathlib import Path
+import subprocess
+import requests
+import ort_test_dir_utils
+import onnxruntime
+import sys
+import os
+import shutil
+import time
+
+cwd_path = Path.cwd()
+TEST_DIR = 'test_dir'
+
+def run_lfs_install():
+    result = subprocess.run(['git', 'lfs', 'install'], cwd=cwd_path, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    print('Git LFS install completed with return code= {}'.format(result.returncode))
+
+
+def pull_lfs_file(file_name):
+    result = subprocess.run(['git', 'lfs', 'pull', '--include', file_name, '--exclude', '\'\''], cwd=cwd_path, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    print('LFS pull completed with return code= {}'.format(result.returncode))
+
+
+def save_file(url, file_name):
+  r = requests.get(url)
+  with open(file_name, 'wb') as f:
+    f.write(r.content)
+
+def check_by_onnx(model_path, model_name):
+    model = onnx.load(model_path)
+    onnx.checker.check_model(model)
+    print('[PASS]: {} is checked by onnx. '.format(model_name))
+
+def check_by_onnxruntime(model_path, model_name):
+    onnxruntime.InferenceSession(model_path)
+    ort_test_dir_utils.create_test_dir(model_path, './', TEST_DIR)
```

Comment:
Model zoo has a zipped folder along with every model which contains the sample test data... you should use that instead as we need to validate the test data as well